frontend/express/public/core/events/javascripts/countly.details.views.js

```diff
@@ -262,42 +302,45 @@
             lineLegend: function() {
                 return this.$store.getters["countlyAllEvents/legendData"];
             },
-            allEvents: function() {
-                var list = this.$store.getters["countlyAllEvents/allEvents"].list;
-                if (list) {
-                    return list.map(function(item) {
-                        return {
-                            "label": item,
-                            "value": item
-                        };
-                    });
-                }
-                return list;
-            },
             searchPlaceholder: function() {
-                var list = this.$store.getters["countlyAllEvents/allEvents"].list;
+                var list = this.$store.getters["countlyAllEvents/allEventsList"];
                 if (list) {
                     return this.i18n("events.all.search.placeholder", list.length);
                 }
                 return this.i18n("events.all.search.placeholder");
+            },
+            allEvents: function() {
+                return this.$store.getters["countlyAllEvents/allEventsList"];
+            },
+            eventDescription: function() {
+                return this.$store.getters["countlyAllEvents/allEventsProcessed"].eventDescription;
```

Comment:
are you sure, this.$store.getters["countlyAllEvents/allEventsProcessed"] has a eventDescription key?
what is the structure of this.$store.getters["countlyAllEvents/allEventsProcessed"] ?